File: APIManga/Model/Genre.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace APIManga.Model
{
	public class Genre
	{
		[Key]
		public int Id { get; set; }
		public string Name { get; set; }
		public virtual List<Manga> Mangas { get; set; }
	}
}

File: backend/app/models/genre.py
from sqlalchemy import Column, Integer, String
from sqlalchemy.orm import relationship

from app.database.connection import Base

class Genre(Base):
    __tablename__ = "genre"

    id = Column(Integer, primary_key=True)
    name = Column(String, nullable=False)

    # RELAÇÕES
    mangas = relationship(
        "Manga",
        secondary="manga_genre",
        back_populates="genres"
    )